optaplanner-persistence/optaplanner-persistence-jpa/src/test/java/org/optaplanner/persistence/jpa/impl/AbstractScoreJpaTest.java

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package org.optaplanner.persistence.jpa;
+package org.optaplanner.persistence.jpa.impl;
```

Comment:
Although I don't have anything against this change, I don't see how it's related to the stated goal of this PR.

Reply:
It comes from the same commit as the rest of the changes, so it will stay like that in the git history. Could we at least make it a separate commit?

Reply:
Sorry about that. Next time I'll make it a separate commit, but sorting that out now for a git history that few people look at, isn't worth the hassle, I 'd argue.